packages/expo-firebase-analytics/README.md

```diff
@@ -0,0 +1,425 @@
+
+# API documentation
+
+- [Documentation for the master branch](https://github.com/expo/expo/blob/master/docs/pages/versions/unversioned/sdk/firebase-analytics.md)
+
+# Installation in managed Expo projects
+
+For managed [managed](https://docs.expo.io/versions/latest/introduction/managed-vs-bare/) Expo projects, please follow the installation instructions in the [API documentation for the latest stable release](#api-documentation). If you follow the link and there is no documentation available then this library is not yet usable within managed projects &mdash; it is likely to be included in an upcoming Expo SDK release.
+
+# Installation in bare React Native projects
+
+For bare React Native projects, you must ensure that you have [installed and configured the `react-native-unimodules` package](https://github.com/unimodules/react-native-unimodules) before continuing.
+
+### Add the package to your npm dependencies
+
+```
+expo install expo-firebase-analytics
+```
+
+# Contributing
+
+Contributions are very welcome! Please refer to guidelines described in the [contributing guide]( https://github.com/expo/expo#contributing).
+
+
+**Optional: Enable AdSupport (Bare Workflow)**
+
+To get extra features like `audiences`, `campaign attribution`, and some `user properties`, such as `Age` and `Interests`, you will need to include AdSupport.
+This isn't enabled by default because Apple & Google are strict with allowing apps to use this library.
+
+To enable the AdSupport framework:
+
+- In your Xcode project, select your project's target
+- Select the General tab for your target
+- Expand the Linked Frameworks and Libraries section
+- Click `+` to add a framework
+- Select `AdSupport.framework`
+
+[Learn more in the Firebase Docs](https://firebase.google.com/support/guides/analytics-adsupport)
+
+# Docs
+
+```js
+import * as Analytics from 'expo-firebase-analytics';
+```
+
+To use web analytics, you'll also need to install the peer dependency **firebase** with `yarn add firebase`.
+
+## Methods
+
+### initializeAppDangerously
+
+```tsx
+initializeAppDangerously(googleServices: { [key: string]: any }): Promise<void>
+```
+
+Similar to `firebase.initializeApp()` on web but works to start a native Firebase app while the app is running.
+This can be used to test the native iOS Firebase app in the Expo client.
+This method should not be used in production, instead the app should be bundled with the native Google Services files via the `app.json`.
+
+> You can convert your `GoogleServices-info.plist` using `npx plist-to-json ./GoogleServices-info.plist`.
+
+#### Parameters
+
+| Name       | Type   | Description                                                      |
+| ---------- | ------ | ---------------------------------------------------------------- |
+| name       | string | Platform specific Google Services file for starting a Firebase app during runtime  |
+
+#### Example
+
+```js
+if (Platform.OS === 'ios' && global.__DEV__ === true) {
```

Comment:
`__DEV__` checks will make it difficult to test app in production mode in Expo client, but I don't have any better idea for that (had to face this when developing `expo-notifications` too). Do you think there is a better way to do this check?